api/src/main/scala/dcos/metronome/api/RestController.scala

```diff
@@ -2,22 +2,22 @@ package dcos.metronome
 package api
 
 import com.eclipsesource.schema.SchemaValidator
-import com.wix.accord.{ Validator, Success, Failure }
+import com.wix.accord.{ Failure, Success, Validator }
```

Comment:
There are several changes here which are just reordering of imports... is there a reason?

Reply:
It is IDEA. +1 to the alphabetical ordering

Reply:
Yes. Just IDEA reordering it

Reply:
I'll mark as resolve and keep the change. Feel free to comment again if that is not the desired result.